scim/data_groups_test.go

```diff
@@ -0,0 +1,97 @@
+package scim
+
+import (
+	"testing"
+
+	"github.com/databricks/terraform-provider-databricks/qa"
+)
+
+func TestDataSourceGroups_Error(t *testing.T) {
+	qa.ResourceFixture{
+		Fixtures:    qa.HTTPFailures,
+		Resource:    DataSourceGroups(),
+		Read:        true,
+		NonWritable: true,
+		ID:          "_",
+	}.ExpectError(t, "i'm a teapot")
+}
+
+func TestDataSourceGroups(t *testing.T) {
+	qa.ResourceFixture{
+		Fixtures: []qa.HTTPFixture{
```

Comment:
It's better to use Mock framework: https://github.com/databricks/terraform-provider-databricks/blob/main/catalog/data_external_location_test.go#L14

Reply:
Done